layersvt/vulkan_json_layer.hpp

```diff
@@ -783,6 +790,13 @@ class PipelineData {
     void dumpObjectReservationInfo()
     {
         std::string filename = getenv("VK_JSON_FILE_PATH");
+        if (!filename.empty()) {
```

Comment:
We've got different logic here for the getenv line that we do above. I'm not sure if the NULL check is actually needed, but we might as well move this logic into a `getJsonBasePath` function (or something like that) so we don't need to duplicate it.

Reply:
Replaced all instances of this logic with calls to a new function getBaseDirectoryPath